src/netaddress.cpp

```diff
@@ -70,6 +70,7 @@ bool CNetAddr::SetSpecial(const std::string &strName)
 CNetAddr::CNetAddr(const struct in_addr& ipv4Addr)
 {
     SetRaw(NET_IPV4, (const uint8_t*)&ipv4Addr);
+    scopeId = 0;
```

Comment:
I'd go a step further and move `scopeId = 0` to static initialization instead:
```
    uint32_t scopeId{0}; // for scoped/link-local ipv6 addresses
```

Reply:
Good point! Switched to in-class initialization.